components/director/pkg/accessstrategy/cmp_mtls_access_strategy.go

```diff
@@ -20,22 +20,26 @@ type HTTPRoundTripper interface {
 const tenantHeader = "tenant"
 
 type cmpMTLSAccessStrategyExecutor struct {
-	certCache          certloader.Cache
-	tenantProviderFunc func(ctx context.Context) (string, error)
+	certCache                    certloader.Cache
+	tenantProviderFunc           func(ctx context.Context) (string, error)
+	externalClientCertSecretName string
+	extSvcClientCertSecretName   string
```

Comment:
We could be consistent in the naming 'externalClientCertSecretName' and 'externalSvcClientCertSecretName'

Reply:
`extSvc `stands for `extention service`